test/testbot.js

```diff
@@ -0,0 +1,17 @@
+const Bot = require('scuttle-testbot')
+
+module.exports = function Testbot (opts = {}) {
+  const stack = Bot
+    // NOTE: these already already in scuttlebot
+    // .use(require('ssb-db2'))
+    // .use(require('ssb-box2'))
+    .use(require('../'))
```

Comment:
@mixmix I know you like this tool, but I don't think it's helping us in this case. I doesn't include ssb-box2, as far as I can see.

Reply:
Also, we can't get to choose the version of ssb-db2 (e.g. version 6).

Reply:
Yeah I'm fine too pull it out and copy in the functionality I want from that module. I like the pattern of having the tesbot util. I have some strong opinions about writing ssb tests

Namely - start and stop a server for each test. The main reason is keeping the database clean and clear makes running  standalone  tests easy with test.only, and you never accidentally make graphs of coupled tests by leaving a mess of messages which effect many tests

Reply:
Agree with the second paragraph!

Reply:
@staltz you mean avoiding `scuttle-testbot` entirely? What's the alternative?

Reply:
@Powersource Yeah perhaps you can do like this PR did: https://github.com/ssbc/ssb-meta-feeds/pull/73

Reply:
@staltz done

Reply:
I did this final step